app/controllers/stripe-setup/responsible-person/post.controller.js

```diff
@@ -24,6 +31,8 @@ const HOME_ADDRESS_POSTCODE_FIELD = 'home-address-postcode'
 const DOB_DAY_FIELD = 'dob-day'
 const DOB_MONTH_FIELD = 'dob-month'
 const DOB_YEAR_FIELD = 'dob-year'
+const TELEPHONE_NUMBER_FIELD = 'telephone'
```

Comment:
Can we call this `telephone-number`, just to be consistent with other data we are collecting at different pages?